interactive_engine/executor/ir/integrated/tests/pathxd_test.rs

```diff
@@ -625,4 +625,205 @@ mod test {
     fn path_expand_with_filter_query_w2_test() {
         path_expand_with_filter_query(2)
     }
+
+    // both(1..3) with vertices and edges preserved in the path
+    fn path_expand_whole_v_e_query(worker_num: u32) {
+        initialize();
+        let request = init_path_expand_request(pb::Range { lower: 1, upper: 3 }, 2, 0);
+        let mut results = submit_query(request, worker_num);
+        let mut result_collection: Vec<Vec<ID>> = vec![];
+        let mut expected_result_paths = vec![
+            // v1, e[1->2], v2
+            vec![1, 1, 2, 2],
```

Comment:
I think making the element `String`, and if it is a vertex, turn ID into String, and if it is an Edge, turn (ID, ID) into string. This will make the results more clear.

Reply:
Done.